### ISSUE TITLE
A question about loss function

### ISSUE THREAD

**Issue by JiangWeibeta** [Lornatang/WassersteinGAN_DIV-PyTorch#2]
D_loss , errD_real is  "-torch.mean(fake_output)" and errG is changed from "-torch.mean(fake_output)" to "torch.mean(fake_output)" ， which is different from Pytorch-GAN, can you tell me the reason?

**Comment by Lornatang** [comment]
please tell me the source of reference.
